### ISSUE TITLE
Paramiko missing on oob-mgmt-server for Ansible

### ISSUE THREAD

**Issue by rdarbha** [CumulusNetworks/topology_converter#133]
```
cumulus@oob-mgmt-server:~/yahoojp/automation$ ansible-playbook automate.yml -l lfc-1am01 -t netq_agent
 [WARNING]: Invalid characters were found in group names but not replaced, use -vvvv to see details


PLAY [Configure Chassis Line Cards] ***********************************************************************************************************************************************
skipping: no hosts matched

PLAY [Configure Chassis Fabric Cards] *********************************************************************************************************************************************
skipping: no hosts matched

PLAY [Configure Leaf Switches] ****************************************************************************************************************************************************

TASK [netq : Apply Netq repos] ****************************************************************************************************************************************************
Friday 12 July 2019  08:09:02 -0700 (0:00:00.088)       0:00:00.088 ***********
fatal: [lfc-1am01]: FAILED! => {"msg": "paramiko is not installed: No module named 'paramiko'"}

PLAY RECAP ************************************************************************************************************************************************************************
lfc-1am01                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

Friday 12 July 2019  08:09:02 -0700 (0:00:00.030)       0:00:00.119 ***********
===============================================================================
netq : Apply Netq repos ---------------------------------------------------------------------------------------------------------------------------------------------------- 0.03s
cumulus@oob-mgmt-server:~/yahoojp/automation$ pip freeze
argcomplete==1.8.1
asn1crypto==0.24.0
cryptography==2.1.4
enum34==1.1.6
idna==2.6
ipaddr==2.2.0
ipaddress==1.0.17
keyring==10.6.0
keyrings.alt==3.0
pycrypto==2.6.1
pygobject==3.26.1
pyxdg==0.25
SecretStorage==2.3.1
six==1.11.0
```

This fixes it:
`cumulus@oob-mgmt-server:~/yahoojp/automation$ sudo pip3 install paramiko`

**Comment by jubetz** [comment]
some automation needs netaddr library as well. Could squeeze it in with this 

`pip3 install netaddr`

**Comment by ericpulvino** [comment]
Fixed in 8a2f01e
